events/wc_frostwolf_events.txt

```diff
@@ -1086,34 +1127,289 @@ narrative_event = { #Frostwolf settled
 	}
 	option = {
 		trigger = {
-			NOT = { character = ROOT }
+			NOT = { character = event_target:target_frostwolf_leader }
 			culture_group = orc_group
 		}
-		name = EVTOPTC_WCFW.45 #These rats cannot hide from the Horde!
+		name = EVTOPTC_WCFW_50 # Rats! The Horde will hunt them all!
 		piety = 5
 	}
 }
 
-# FAIL EVENT
-character_event = {
-	id = WCFW.200
-	picture = GFX_evt_orcish_horde
-	desc =  EVTDESC_WCFW_200
-	border = GFX_event_normal_frame_war
+# Frostwolf failure event 60-63
+# delayer event
+province_event = {
+	id = WCFW.160
 	
 	is_triggered_only = yes
+	hide_window = yes
 	
 	immediate = {
 		d_frostwolf = {
-			clr_title_flag = fw_hunger_event_flag
+			holder_scope = {
+				if = {
+					limit = { frostwolf_leader_exists = yes }
+					narrative_event = { id = WCFW.60 }
+				}
+			}
 		}
 	}
+}
```

Comment:
There's no need in this event. Firing of this event can be replaced by `WCFW.60` because `WCFW.160` fires without delay.

Reply:
The reason why I did this was because I felt it was "nicer" to write the whole `d_frostwolf` and `holder_scope` block just once instead of putting it in every reference to `WCFW.160`... Since if I remove this block, I'll have to put the `holder_scope` in all the other events so it reaches the Frostwolf leader, no?

Reply:
Mmmkay. Fine then.